app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -3,31 +3,46 @@ import React from "react";
 import { MenuVertical } from "neetoicons";
 import { Avatar, Typography, Dropdown } from "neetoui";
 
-const renderAvatarWithNameRoleAndImage = (name, { role, imageUrl }) => (
+const renderAvatarWithNameRoleAndImage = (
+  name,
+  { firstName, lastName, role, imageUrl }
+) => (
   <div className="flex flex-row items-center">
     <Avatar
       className="mr-2"
       user={{
-        name,
+        name: `${firstName} ${lastName}`,
         imageUrl,
       }}
     />
     <div>
-      <Typography style="h4">{name}</Typography>
+      <Typography style="h4">{`${firstName} ${lastName}`}</Typography>
       <Typography style="body2">{role}</Typography>
     </div>
   </div>
 );
 
-const renderDropdownButton = (setShowDeleteAlert, setSelectedContactId, id) => (
+const handleDelete = ({ setShowDeleteAlert, setSelectedContactId, id }) => {
+  setShowDeleteAlert(true);
+  setSelectedContactId(id);
+};
+
+const renderDropdownButton = ({
+  setShowDeleteAlert,
+  setSelectedContactId,
+  id,
+}) => (
   <div>
     <Dropdown buttonStyle="text" icon={MenuVertical}>
       <li> Edit </li>
       <li
-        onClick={() => {
-          setShowDeleteAlert(true);
-          setSelectedContactId(id);
-        }}
+        onClick={() =>
+          handleDelete({
+            setShowDeleteAlert,
+            setSelectedContactId,
+            id,
+          })
```

Comment:
If there's no use case of destructuring the props, then don't do it. Here you only need to pass these props down the line.
```suggestion
const renderDropdownButton = props => (
  <div>
    <Dropdown buttonStyle="text" icon={MenuVertical}>
      <li> Edit </li>
      <li
        onClick={() =>
          handleDelete(props)
```